conu/apidefs/backend.py

```diff
@@ -99,13 +99,14 @@ class Backend(object):
     ContainerClass = Container
     ImageClass = Image
 
-    def __init__(self, logging_level=logging.INFO, logging_kwargs=None):
+    def __init__(self, logging_level=logging.INFO, logging_kwargs=None, cleanup=True):
```

Comment:
would it be possible to rewrite this in a similar manner as https://github.com/fedora-modularity/conu/pull/131/files#diff-2506762216f23d81a6f649c275db50bdR53 so that it's a policy and users will be able to pick more than just 2 states; @dhodovsk WDYT?

Reply:
I agree that there could be more states for cleanup configurations, or types of resources to be cleaned up (list? O.o). We could come up with few possible values of cleanup policy that could be implemented now or in future.
Any suggestions?

Reply:
* clean everything
* clean nothing
* clean containers
* clean volumes
* clean images
* clean temporary directories

(obviously, it's out of scope of this PR to implement them)

Reply:
So it could be a list then? Something like
`cleanup=[CleanupPolicy.volumes, ClueanupPolicy.containers]`

Reply:
sounds good